internal/nginx/maxmind.go

```diff
@@ -81,6 +103,17 @@ func GeoLite2DBExists() bool {
 	return true
 }
 
+// DownloadGeoLite2DBPeriodically starts a goroutine to periodically sync the GeoIP databases
+func DownloadGeoLite2DBPeriodically(syncPeriod time.Duration) {
```

Comment:
I thought Maxmind did this itself, which is why it needed curl; we discovered this when we tried to remove Curl and found it was dependent on Maxmind; it setups a cronjob to download.

Reply:
Apologies for the delayed response, I was on holiday. I've tried to look for cronjobs or any other Maxmind related references in this repository, but couldn't find any curl/cronjob used by Maxmind. As far as I understand the only file which downloads db is [internal/nginx/maxmind.go](https://github.com/kubernetes/ingress-nginx/blob/main/internal/nginx/maxmind.go). Could you send me the link where Maxmind setups a cronjob to download a db?

Reply:
@strongjz Hey! Are there any actions I can do to help with review of this PR?

Reply:
@rpkatz found it when we were trying to remove curl from the container image.
 https://alpine.pkgs.org/3.17/alpine-main-aarch64/libmaxminddb-1.7.1-r0.apk.html

One of the deps is curl.